frontend/mr-admin-flate/src/components/lenker/Lenker.tsx

```diff
@@ -0,0 +1,85 @@
+import { Box, Button, HStack, Heading, Switch, TextField, VStack } from "@navikt/ds-react";
+import { useFieldArray, useFormContext } from "react-hook-form";
+import z from "zod";
+import styles from "./Lenker.module.scss";
+import { FaneinnholdSchema } from "../redaksjonelt-innhold/FaneinnholdSchema";
+
+const LenkeSchema = z.object({
+  faneinnhold: FaneinnholdSchema.pick({ lenker: true }),
+});
+type Lenke = z.infer<typeof LenkeSchema>;
```

Comment:
Dette gjør at `<Lenkeskjema />`-komponenten kan brukes for både avtaleskjema og gjennomføringskjema siden faneinnhold er likt på tvers av de to skjemaene.